docs/docs/main/SUMMARY.md

```diff
@@ -0,0 +1,8 @@
+- [About](about/)
+- [Get Started](getting-started/)
+- [Developer Guide](developer-guide/)
+- [Tutorials](examples/)
+- [Data Sets](datasets/)
+- [Contributing](contributing/)
+- [References](references/)
+- [Resources](resources/)
```

Comment:
There is no `resources/` - this yields 404 in the built docs:

![image](https://github.com/user-attachments/assets/691828c4-07b0-452f-bc86-eeace36f36a3)